src/modules/licence-import/load/connectors/queries.js

```diff
@@ -110,8 +110,22 @@ ON CONFLICT (company_id, address_id, role_id) DO UPDATE SET
 exports.createAgreement = `
 INSERT INTO water.licence_agreements (licence_ref, financial_agreement_type_id, start_date, end_date, date_created, date_updated)
 VALUES ($1, $2, $3, $4, NOW(), NOW())
-ON CONFLICT (licence_ref, financial_agreement_type_id, start_date) DO UPDATE SET 
+ON CONFLICT (licence_ref, financial_agreement_type_id, start_date) DO UPDATE SET
   end_date=EXCLUDED.end_date,
   date_updated=EXCLUDED.date_updated
-`
-;
+`;
+
+exports.createLicence = `
+  insert into water.licences (region_id, licence_ref, is_water_undertaker, regions)
+  values (
+    (select region_id from water.regions where nald_region_id = $1),
+    $2,
+    $3,
+    $4
+  )
+  on conflict (region_id, licence_ref) do update
```

Comment:
As with water-service, I think just looking for `on conflict (licence_ref)` should be sufficient